FEWD-Assignment3/FEWD-Assignment3.css

```diff
@@ -0,0 +1,290 @@
+*
+Project Name: Relaxr Landing Page
+Client: Relaxr
+Author: Thalia Wiggins
+Developer @GA in DC
+*/
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+    -moz-box-sizing: border-box; /* Firexfox */
+    -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+    box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+    content: ".";
+    display: block;
+    clear: both;
+    visibility: hidden;
+    line-height: 0;
+    height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+.alignright {
+ float: right;
+ padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+ float: left;
+ padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLE
+/*******************************************/
+
+body {
+    background-color: #f0efef
+    font-size: 12px;
```

Comment:
You're missing a `;` after `background-color: #f0efef` which is why the `font-size: 12px;` is miscolored.